src/logstash/pipeline/sync_production_locations.conf

```diff
@@ -88,6 +88,9 @@ filter {
   ruby {
     path => "/usr/share/logstash/scripts/production_locations/historical_os_id.rb"
   }
+  ruby {
+    path => "/usr/share/logstash/scripts/production_locations/additional_identifiers.rb"
```

Comment:
Since the IDs shouldn’t be covered by the `additional_identifiers` key, please create separate filters for each identifier.